### ISSUE TITLE
go install is broken

### ISSUE THREAD

**Issue by tibers** [boxboat/grypeadmissioncontroller#40]
from @sudo-bmitch 

```
$ go install github.com/boxboat/grypeadmissioncontroller@v1.1.0
go: github.com/boxboat/grypeadmissioncontroller@v1.1.0: github.com/boxboat/grypeadmissioncontroller@v1.1.0: invalid version: unknown revision v1.1.0

$ go install github.com/boxboat/grypeadmissioncontroller@1.1.0
go: downloading github.com/boxboat/grypeadmissioncontroller v0.0.0-20230220181733-1edb12570446
go: github.com/boxboat/grypeadmissioncontroller@1.1.0: github.com/boxboat/grypeadmissioncontroller@v0.0.0-20230220181733-1edb12570446: parsing go.mod:
        module declares its path as: grypy
                but was required as: github.com/boxboat/grypeadmissioncontroller
```

**Comment by tibers** [comment]
```
➜  grypeadmissioncontroller git:(main) rm go.* ; go mod init grypeadmissioncontroller ; go mod tidy; go mod vendor
go: creating new go.mod: module grypeadmissioncontroller
go: copying requirements from Gopkg.lock
go: to add module requirements and sums:
	go mod tidy
go: finding module for package k8s.io/api/admission/v1
go: finding module for package github.com/stretchr/testify/require
go: finding module for package github.com/davecgh/go-spew/spew
go: finding module for package github.com/stretchr/testify/assert
go: downloading k8s.io/api v0.26.2
go: downloading github.com/stretchr/testify v1.8.2
go: found k8s.io/api/admission/v1 in k8s.io/api v0.26.2
go: found github.com/stretchr/testify/assert in github.com/stretchr/testify v1.8.2
go: found github.com/stretchr/testify/require in github.com/stretchr/testify v1.8.2
go: found github.com/davecgh/go-spew/spew in github.com/davecgh/go-spew v1.1.1
go: downloading k8s.io/apimachinery v0.26.2
➜  grypeadmissioncontroller git:(main) ✗ git add .
➜  grypeadmissioncontroller git:(main) ✗ git commit -am 'last try to fix this for bmitch'
[main 0e6cb6a] last try to fix this for bmitch
 11 files changed, 955 insertions(+), 1617 deletions(-)
 create mode 100644 vendor/github.com/golang/glog/README
 delete mode 100644 vendor/github.com/golang/glog/README.md
 delete mode 100644 vendor/github.com/golang/glog/glog_flags.go
 delete mode 100644 vendor/github.com/golang/glog/internal/logsink/logsink.go
 delete mode 100644 vendor/github.com/golang/glog/internal/logsink/logsink_fatal.go
 delete mode 100644 vendor/github.com/golang/glog/internal/stackdump/stackdump.go
➜  grypeadmissioncontroller git:(main) git push
Enumerating objects: 22, done.
Counting objects: 100% (22/22), done.
Delta compression using up to 12 threads
Compressing objects: 100% (11/11), done.
Writing objects: 100% (12/12), 13.30 KiB | 4.43 MiB/s, done.
Total 12 (delta 6), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (6/6), completed with 6 local objects.
To github.com:boxboat/grypeadmissioncontroller.git
   6de18c8..0e6cb6a  main -> main
➜  grypeadmissioncontroller git:(main) go install github.com/boxboat/grypeadmissioncontroller@main
go: github.com/boxboat/grypeadmissioncontroller@main: github.com/boxboat/grypeadmissioncontroller@v1.1.2-0.20230303154201-370c6c6be341: parsing go.mod:
	module declares its path as: grypeadmissioncontroller
	        but was required as: github.com/boxboat/grypeadmissioncontroller
```

**Comment by tibers** [comment]
Closed by v1.1.2
